pkg/identityserver/user_registry.go

```diff
@@ -310,6 +309,24 @@ func (is *IdentityServer) updateUser(ctx context.Context, req *ttnpb.UpdateUserR
 		cleanContactInfo(req.User.ContactInfo)
 	}
 
+	if ttnpb.HasAnyField(req.FieldMask.Paths, "temporary_password") {
+		hashedTemporaryPassword, err := auth.Hash(ctx, req.User.TemporaryPassword)
+		if err != nil {
+			return nil, err
+		}
+		req.User.TemporaryPassword = hashedTemporaryPassword
+		now := time.Now()
+		if !ttnpb.HasAnyField(req.FieldMask.Paths, "temporary_password_created_at") {
```

Comment:
Why allow setting this by the user?
IMO registries should fully manage the `created_at`/`updated_at` fields, hence set this to `now` always.(that's also the behavior of all other registries in `master` - `.*updated_at` and `.*created_at` fields are not allowed to be set)

Reply:
1. It's an admin doing this, not a user.
2. Disallowing this now would break external API.